### DISCUSSION TITLE
TermStore GetTermSetByIdAsync method not found

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#906] [Q&A] [answered]
Hello,
In documentation here [Taxonomy intro](https://pnp.github.io/pnpcore/using-the-sdk/taxonomy-intro.html) is written that interface `ITermStore` should implement method `GetTermSetById`, but in my code i can't see that method
![image](https://user-images.githubusercontent.com/42325625/176404121-0d3cc156-4b4e-4c6e-8e05-03b9012f0ad9.png)

PnP.Core version used: 1.6.0
in which version it is implemented? currently i 1.6.0 is newest possible that VS2022 is giving me as an option

Thank you for your answer

**Comment by gszdev** [comment, marked as answer]
@Nevinia210 as @jansenbe mentationed in #892 [Comment](https://github.com/pnp/pnpcore/issues/892#issuecomment-1168802199) you need the nighly build or wait for the 1.7 release comming this week.
